pyrate/core/ifgconstants.py

```diff
@@ -34,6 +34,23 @@
 PYRATE_TIME_SPAN = 'TIME_SPAN_YEAR'
 PYRATE_WAVELENGTH_METRES = 'WAVELENGTH_METRES'
 PYRATE_INCIDENCE_DEGREES = 'INCIDENCE_DEGREES'
+PYRATE_HEADING_DEGREES = 'HEADING_DEGREES'
+PYRATE_RANGE_PIX_METRES = 'RANGE_PIX_METRES'
+PYRATE_RANGE_N = 'RANGE_N'
+PYRATE_AZIMUTH_PIX_METRES = 'AZIMUTH_PIX_METRES'
+PYRATE_AZIMUTH_N = 'AZIMUTH_N'
+PYRATE_AZIMUTH_LOOKS = 'AZIMUTH_LOOKS'
```

Comment:
Is it worth adding RANGE_LOOKS for completeness of information, even if it is not used for a calculation?

Reply:
see https://github.com/GeoscienceAustralia/PyRate/pull/302/commits/fa2a65850f4aec5bfc26712c9339f19b6edc51ee